index.html

```diff
@@ -0,0 +1,171 @@
+    </div>
+    <section class="program-list">
+      <div class="programs">
+        <div>
+          <img class="mt-4" src="images/self-leadership.png" alt="self">
+        </div>
+        <div class="program-body">
+          <div class="program-title">
+            <h4 class="card-title">Self-leadership</h4>
+          </div>
+          <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing.</p>
+        </div>
+      </div>
+      <div class="programs">
+        <div>
+          <img class="mt-4" src="images/servant-leadership.png" alt="servant">
+        </div>
+        <div class="program-body">
+          <div class="program-title">
+            <h4 class="card-title">Servant leadership</h4>
+          </div>
+          <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing.</p>
+        </div>
+      </div>
+      <div class="programs">
+        <div>
+          <img class="mt-4" src="images/program-icon-03.png" alt="teams">
+        </div>
+        <div class="program-body">
+          <div class="program-title">
+            <h4 class="card-title">Team leadership</h4>
+          </div>
+          <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing.</p>
+        </div>
+      </div>
+      <div class="programs">
+        <div>
+          <img class="mt-4" src="images/program-icon-04.png" alt="community">
+        </div>
+        <div class="program-body">
+          <div class="program-title">
+            <h4 class="card-title">Organisation change</h4>
+          </div>
+          <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing.</p>
+        </div>
+      </div>
+      <div class="programs">
+        <div>
+          <img class="mt-3" src="images/program-icon-05.png" alt="transformation">
+        </div>
+        <div class="program-body">
+          <div class="program-title">
+            <h4 class="card-title">Transform project</h4>
+          </div>
+          <p class="program-text">Lorem ipsum dolor sit amet consectetur adipisicing.</p>
+        </div>
+      </div>      
+    </section>
+    <button class="all-programs" type="button">
+        See All Programs
+      </button>
+  </section>
+
+  <!--Leaders-->
+  <section class="leaders" id="leaders">    
+  </section>
+
+  <!--Partners-->
+  <section class="partners" id="partners">
+    <h4 class="partner-heading">
+      Partners
+    </h4>
+    <div class="partner-block ms-5">
+      <div class="part1">
+        <div>
+          <img class="partner1" src="images/power.png" alt="">
+        </div>
+        <div>
+          <img class="partner2" src="images/Hope.png" alt="">
+        </div>
+      </div>
+      <div class="part2">
+        <div>
+          <img class="partner3" src="images/futuristic.png" alt="futuristic">
+        </div>
+        <div>
+          <img class="partner4" src="images/socl-observer.png" alt="observer">
+        </div>
+      </div>
+    </div>
+  </section>
```

Comment:
Kindly try to match the partner section in the desktop version to match the Behance design.
| Your design | Behance design | 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/157788700-3ef3316e-3dca-4020-a325-1e2f0b2291fe.png) | ![image](https://user-images.githubusercontent.com/24703171/157788745-cf648f08-8c4c-4e45-8772-5498306b275c.png) |

Reply:
Addressed!